tests/launch_tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
For some reason, when I cloned your fork, the `launch_tests.sh` file and the `tests.py` file didn't have the executable permission (but weirdly the asciinema showed them to be executable; no idea what happened there). Mind adding `chmod +x` to the files? :)